test-cases/manifest.ttl

```diff
@@ -1,3338 +1,4844 @@
-  rmltest:hasExpectedOutput true;
-  rmltest:input [
-      rmltest:inputFile "student.csv";
-      rmltest:inputType "CSV"
-    ];
+  rmltest:hasError false;
+  rmltest:input <http://w3id.org/rml/core/test/RMLTC0000-CSV/input/http%3A%2F%2Fw3id.org%2Frml%2Fresources%2Frml-io%2FRMLTC0000-CSV%2FFriends.json/>;
   rmltest:mappingDocument "mapping.ttl";
-  rmltest:output "output.nq" .
+  rmltest:output <http://w3id.org/rml/core/test/RMLTC0000-CSV/output/output.nq/> .
 
-<RMLTC0000-JSON> a test:TestCase;
+<http://w3id.org/rml/core/test/RMLTC0000-CSV/input/http%3A%2F%2Fw3id.org%2Frml%2Fresources%2Frml-io%2FRMLTC0000-CSV%2FFriends.json/>
+  a rmltest:Input;
+  rmltest:input "http://w3id.org/rml/resources/rml-io/RMLTC0000-CSV/Friends.json";
+  rmltest:inputFormat "application/json" .
+
+<http://w3id.org/rml/core/test/RMLTC0000-CSV/output/output.nq/> a rmltest:Output;
+  rmltest:output "output.nq";
+  rmltest:outputFormat "application/n-quads" .
+
+<http://w3id.org/rml/core/test/RMLTC0000-JSON> a test:TestCase;
   dcterms:identifier "RMLTC0000-JSON";
-  rmltest:hasExpectedOutput true;
-  rmltest:input [
-      rmltest:inputFile "student.json";
-      rmltest:inputType "JSON"
-    ];
+  rmltest:hasError false;
+  rmltest:input <http://w3id.org/rml/core/test/RMLTC0000-JSON/input/http%3A%2F%2Fw3id.org%2Frml%2Fresources%2Frml-io%2FRMLTC0000-JSON%2FFriends.json/>;
   rmltest:mappingDocument "mapping.ttl";
-  rmltest:output "output.nq" .
+  rmltest:output <http://w3id.org/rml/core/test/RMLTC0000-JSON/output/output.nq/> .
+
+<http://w3id.org/rml/core/test/RMLTC0000-JSON/input/http%3A%2F%2Fw3id.org%2Frml%2Fresources%2Frml-io%2FRMLTC0000-JSON%2FFriends.json/>
+  a rmltest:Input;
+  rmltest:input "http://w3id.org/rml/resources/rml-io/RMLTC0000-JSON/Friends.json";
+  rmltest:inputFormat "application/json" .
+
+<http://w3id.org/rml/core/test/RMLTC0000-JSON/output/output.nq/> a rmltest:Output;
+  rmltest:output "output.nq";
+  rmltest:outputFormat "application/n-quads" .
 
-<RMLTC0000-MySQL> a test:TestCase;
+<http://w3id.org/rml/core/test/RMLTC0000-MySQL> a test:TestCase;
   dcterms:identifier "RMLTC0000-MySQL";
-  rmltest:hasExpectedOutput true;
-  rmltest:input [
-      rmltest:database [
-          rmltest:sqlScriptFile "resource.sql"
-        ];
-      rmltest:inputType "MySQL"
-    ];
+  rmltest:hasError false;
+  rmltest:input <http://w3id.org/rml/core/test/RMLTC0000-MySQL/input/http%3A%2F%2Fw3id.org%2Frml%2Fresources%2Frml-io%2FRMLTC0000-MySQL%2FFriends.json/>;
   rmltest:mappingDocument "mapping.ttl";
-  rmltest:output "output.nq" .
+  rmltest:output <http://w3id.org/rml/core/test/RMLTC0000-MySQL/output/output.nq/> .
 
-<RMLTC0000-PostgreSQL> a test:TestCase;
+<http://w3id.org/rml/core/test/RMLTC0000-MySQL/input/http%3A%2F%2Fw3id.org%2Frml%2Fresources%2Frml-io%2FRMLTC0000-MySQL%2FFriends.json/>
+  a rmltest:Input;
+  rmltest:input "http://w3id.org/rml/resources/rml-io/RMLTC0000-MySQL/Friends.json";
+  rmltest:inputFormat "application/json" .
```

Comment:
is the input JSON for MySQL tests?

Reply:
No, will be the issue of your previous comment, will look into it.